amr-wind/wind_energy/ABLStats.cpp

```diff
@@ -85,6 +86,7 @@ void ABLStats::calc_averages()
     m_pa_tu();
     m_pa_uu();
     m_pa_uuu();
+    calc_sfs_stress_avgs();
```

Comment:
These don't need to be calculated at every timestep... only if we have to output at a particular time instance correct? 

For that matter, the same applies to second and third moment averaging instances too.

Reply:
Tried addressing in https://github.com/Exawind/amr-wind/pull/290/commits/22cbdfe875f7c910a3fd1369c871a361c06ac0dd